zstd-safe/zstd-sys/build.rs

```diff
@@ -152,7 +166,9 @@ fn compile_zstd() {
      * 7+: events at every position (*very* verbose)
      */
     #[cfg(feature = "debug")]
-    config.define("DEBUGLEVEL", Some("5"));
+    if is_wasm_unknown_unknown {
```

Comment:
I think you want the opposite here?
```suggestion
    if !is_wasm_unknown_unknown {
```